CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## Unreleased
+
+### Bug Fixes
+
+- Fix a bug that caused the compiler to crash when accessing a field of a const record, that had a function param in a guard clause.
+  ([Neil Wadden](https://github.com/Neilerino))
+
```

Comment:
I wasn't super sure about this: where another release version was cut I added a new section for `unreleased`, and `bug fixes`. Lemme know if this wasn't correct though, and I'll fix it 👍

Reply:
Yeah that's correct. There's just been a release, so you needed to add that. Thanks!